NEWS.md

```diff
@@ -25,6 +25,9 @@ Language changes
 Compiler/Runtime improvements
 -----------------------------
 
+* The package precompilation process now saves native code into a "pkgimage", meaning that code generated during the
+  precompilation process will not require compilation after package load, resulting in much faster time to first
+  execution (TTFX), often referred to as time to first plot ([#44527]) ([#47184]).
```

Comment:
```suggestion
* Time to first execution (TTFX, sometimes called time to first plot) is greatly reduced. Package precompilation now
  saves native code into a "pkgimage", meaning that code generated during the precompilation process will not
  require compilation after package load. Use of pkgimages can be disabled via `--pkgimages=no` ([#44527]) ([#47184]).
```

Reply:
@timholy look ok? Perhaps we can finalize in a thread to save unnecessary CI runs

Reply:
Oh, I responded to the PR but I guess I mean I like this version.

Reply:
Got it. Thanks